docs/pages/User/Install/pre-install.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Pre-Install
+weight: 2
+---
+
+### Introduction
+
+Liqo can be installed either in private or local clusters. Its configuration depends on the type of connectivity between the two clusters. Before installing Liqo, you have to consider how your clusters can connect to each other and can peer together.
+
+### Peering Requirements
+
+Liqo requires the following services to be reciprocally accessible to perform cluster peerings:
+
+* **Authentication server** used to authenticate other clusters (i.e. `liqo-auth`).
+* The **Kubernetes API server** you want to peer.
+* **Network gateway** used to establish interconnection between clusters (i.e. `liqo-gateway`)
+
+Those services have to be accessible from the other clusters to peer with them. This may change the way you would like to have them exposed.
+Below it is possible to find some common scenarios that Liqo can handle. Once you identify yours, you can go have to the *table* of each section to find the right values you should specify when installing Liqo using the chart.
+
+### Cloud to cloud
```

Comment:
I do not fully understand this section. Are those parameters set during the *setup* of the cluster, or during the *creation* of the peering? In case of the latter, I would propose to make this more clear in the first part of this document.